jwql/instrument_monitors/common_monitors/edb_telemetry_monitor.py

```diff
@@ -1366,7 +1367,7 @@ def multiday_mnemonic_query(self, mnemonic_dict, starting_time_list, ending_time
                         if product_mnemonic_info.meta['TlmMnemonics'][0]['AllPoints'] == 0:
                             # If both mnemonics are change-only, then we need to translate them both
                             # to all-points.
-                            delta_t = timedelta(seconds=1.)
+                            delta_t = datetime.timedelta(seconds=1.)
```

Comment:
How did this never raise an error before??

Reply:
are we missing errors that are getting logged someplace?

Reply:
Yeah, I don't know 🤷‍♂️